src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator  {
+    public String getArea ();
```

Comment:
```suggestion
    String getArea ();
```
в інтерфейсі методи по дефолту публічні і абстрактні